### PR TITLE
Update fabric-gateway-controller

### PR DESCRIPTION
**Pull request by RomanZavodskikh**
Changelog:

+ Create ControllerRevision if stackVersion is set
+ Added some follow-up small enhancenents

### CODE REVIEW

File: cluster/manifests/fabric-gateway/deployment.yaml [hunk 1]
```diff
@@ -47,6 +47,9 @@ spec:
             {{ if eq .Cluster.ConfigItems.fabric_gateway_controller_allow_all_filters "true" }}
             - -allow-all-filters=true
             {{ end }}
+            {{ if eq .Cluster.ConfigItems.fabric_gateway_controller_enable_versioning "true" }}
+            - -enable-versioning
```
Comment (MustafaSaber) [inline cluster/manifests/fabric-gateway/deployment.yaml:51]: I think `=true` is missing

Comment (RomanZavodskikh) [reply cluster/manifests/fabric-gateway/deployment.yaml:51]: `-enable-versioning` and `-enable-versioning=true` work the same way.

**Comment by RomanZavodskikh** [comment]
:+1:

File: cluster/manifests/fabric-gateway/deployment.yaml (hunk 1 above)
Comment (MustafaSaber) [reply cluster/manifests/fabric-gateway/deployment.yaml:51]: I think explicit is better is this case like `-allow-all-filters=true` but that's a minor

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**